Team417_CENTERSTAGE/src/main/java/org/firstinspires/ftc/team417_CENTERSTAGE/mechanisms/ArmTeleOp.java

```diff
@@ -0,0 +1,216 @@
+package org.firstinspires.ftc.team417_CENTERSTAGE.mechanisms;
+
+import static org.firstinspires.ftc.team417_CENTERSTAGE.baseprograms.BaseOpMode.DUMPER_SERVO_RESET_POSITION;
+import static org.firstinspires.ftc.team417_CENTERSTAGE.baseprograms.BaseOpMode.DUMPER_SERVO_TILT_POSITION;
+
+import com.acmerobotics.dashboard.FtcDashboard;
+import com.acmerobotics.dashboard.telemetry.TelemetryPacket;
+import com.qualcomm.robotcore.hardware.DcMotor;
+import com.qualcomm.robotcore.hardware.Gamepad;
+import com.qualcomm.robotcore.hardware.Servo;
+import com.qualcomm.robotcore.util.ElapsedTime;
+
+import org.firstinspires.ftc.team417_CENTERSTAGE.baseprograms.BaseOpMode;
+
+public class ArmTeleOp {
```

Comment:
Big picture comments (like an overview of what the class is responsible for) are super helpful, often more helpful than low-level comments (like "Used to make sure button inputs are only registered once" for "dpadDownPressed").